migrations/versions/52381cafc36f_add_created_at_field_to_rh_accounts.py

```diff
@@ -0,0 +1,32 @@
+"""add created_at field to rh_accounts
+
+Revision ID: 52381cafc36f
+Revises: 020def9f5b94
+Create Date: 2024-08-28 16:11:42.517790
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '52381cafc36f'
+down_revision = '020def9f5b94'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('rh_accounts', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('created_at', sa.DateTime(), nullable=True))
```

Comment:
A thought after approval, do we need index for this column? if we are going to pull the data from table for metrics creation based on different time and date slices then I think its beneficial?
Maybe BRIN type index? https://www.postgresql.org/docs/current/brin-intro.html

Reply:
+1 Makes sense. I have added index of type `brin` on `created_at` column. Please request to review and merge if no additional change needed. Thanks!